src/components/EditCoursePage/index.jsx

```diff
@@ -157,11 +157,15 @@ class EditCoursePage extends React.Component {
       error = error.concat(courseInfo.error, ' ');
     }
     if (courseOptions.error) {
-      error = error.concat(courseOptions.error);
+      error = error.concat(courseOptions.error, ' ');
+    }
+    if (courseRunOptions.error) {
+      error = error.concat(courseRunOptions.error);
     }
     error = error.trim();
     const showSpinner = !startedFetching || courseInfo.isFetching || courseOptions.isFetching;
-    const showForm = startedFetching && !courseInfo.isFetching && !courseOptions.isFetching;
+    const showForm = !error && startedFetching && !courseInfo.isFetching &&
+      !courseOptions.isFetching;
 
     return (
       <React.Fragment>
```

Comment:
Same comment as above, should we provide next steps to the users in the error message below?